src/main/java/no/nav/security/oidc/test/support/spring/TokenGeneratorConfiguration.java

```diff
@@ -18,7 +18,7 @@
      */
     @Bean
     @Primary
-    OIDCResourceRetriever overrideOidcResourceRetriever() {
+    ProxyAwareResourceRetriever overrideOidcResourceRetriever() {
         return new FileResourceRetriever("/local-login/metadata-selvbetjening.json", "/local-login/metadata-isso.json", "/local-login/jwkset.json");
     }
```

Comment:
Ny versjon har fortsatt ikke støtte for flere typer innlogginger under lokal kjøring av appen...